mne/minimum_norm/resolution_matrix.py

```diff
@@ -116,20 +325,64 @@ def get_point_spread(resmat, src, idx, norm=False):
         Forward Operator.
     src : instance of SourceSpaces
         Source space used to compute resolution matrix.
-    idx : list of int
-        Vertex indices for which PSFs or CTFs to produce.
-    norm : bool
-        Whether to normalise to maximum across all PSFs (default: False).
+    idx : list of int | list of Label
+        Source for indices for which to compute PSFs or CTFs. If mode is None,
+        PSFs/CTFs will be returned for all indices. If mode is not None, the
+        corresponding summary measure will be computed across all PSFs/CTFs
+        available from idx.
+        Can be:
+
+        * list of integers : Compute PSFs/CTFs for all indices to source space
+          vertices specified in idx.
+        * list of Label : Compute PSFs/CTFs for source space vertices in
+          specified labels.
+
+    mode : None | 'mean' | 'max' | 'svd'
+        Compute summary of PSFs/CTFs across all indices specified in 'idx'.
+        Can be:
+
+        * None : Output individual PSFs/CTFs for each specific vertex
+          (Default).
+        * 'mean' : Mean of PSFs/CTFs across vertices.
+        * 'max' : PSFs/CTFs with maximum norm across vertices. Returns the
+          n_comp largest PSFs/CTFs.
+        * 'svd' : SVD components across PSFs/CTFs across vertices. Returns the
+          n_comp first SVD components.
+    n_comp : int
+        Number of PSF/CTF components to return for mode='max' or mode='svd'.
+        Default n_comp=1.
+    norm : None | 'max' | 'norm'
+        Whether and how to normalise the PSFs and CTFs. This will be applied
+        before computing summaries as specified in 'mode'.
+        Can be:
+
+        * None : Use un-normalized PSFs/CTFs (Default).
+        * 'max' : Normalize to maximum absolute value across all PSFs/CTFs.
+        * 'norm' : Normalize to maximum norm across all PSFs/CTFs.
+    return_svd_vars : bool
+        Whether or not to return the explained variances across the specified
+        vertices for individual SVD components. This is only valid if
+        mode='svd'.
+        Default return_svd_vars=False.
 
     Returns
     -------
-    stc: instance of SourceEstimate
-        PSFs as an stc object.
+    stc : instance of SourceEstimate
+        PSFs or CTFs as an STC object.
+        All PSFs/CTFs will be returned as successive samples in one STC object,
+        in the order they are specified in idx. PSFs/CTFs for labels are
+        grouped together.
+    return_svd_vars : 1D array
```

Comment:
It's one variance per component. The SVD (and mean/sum) will be computed across all vertices from all labels (the docstring says "Compute summary of PSFs/CTFs across all indices specified in 'idx'."). If the user wants to get this for individual labels, they'll have to call the function for each label (there is no speed benefit doing it otherwise). I'll illustrate this in the example.

Reply:
Following your comment I now think I should change it to be comparable to extract_label_time_courses, i.e. input a list of labels and output a list for labels (rather than summarise across all vertices from all labels). Labels can easily be added before applying this function, if required.
Do you agree?

Reply:
Not sure, what I wrote was just what I naively expected based on looking at the code. I'd let the illustrative example that you're working on be the guide :)